index.html

```diff
@@ -741,6 +738,40 @@
         </section>
 
         <section>
+          <h2 id='ldpcvget'>HTTP GET</h2>
+          <p>
+            An <a>LDPCv</a> MUST respond to <code>GET Accept: application/link-format</code> as
+            indicated in [[!RFC7089]]
+            <a href='https://tools.ietf.org/html/rfc7089#section-5'>section 5</a> and specified in
+            [[!RFC6690]] <a href='https://tools.ietf.org/html/rfc6690#section-7.3'>section 7.3</a>.
+          </p>
+          <p>
+            An implementation SHOULD respond with the <code>Allow</code> header as outlined in 
+            <a href='#ldpcvoptions'></a>.
+          </p>
+          <p>
+            An implementation MUST respond with a <code>Content-Type</code> header.
```

Comment:
It seems like this is potentially crossing a line of over-specifying basic HTTP expectations. Is there a particular that we would want to call out the `Content-Type` header?

Reply:
resolved